gui/package.json

```diff
@@ -12,36 +12,36 @@
     "@types/c3": "0.7.5",
     "@types/moment-timezone": "0.5.12",
     "@types/vuelidate": "0.7.13",
-    "axios": "0.21.1",
+    "axios": "^0.24.0",
```

Comment:
we should not use ~ or ^ sign. We should fix versions for dependencies

Reply:
Okay, I'll make the change.

Reply:
@kupranay, I have modified the package.json file now. Please review the same.